packages/components/chip/src/Chip.stories.tsx

```diff
@@ -7,11 +8,60 @@ const meta: Meta<typeof Chip> = {
   component: Chip,
 }
 
+type ChipProps = ComponentProps<typeof Chip>
+
+const designs: ChipProps['design'][] = ['outlined', 'filled', 'dashed', 'tinted']
+
+const intents: ChipProps['intent'][] = [
+  'primary',
+  'secondary',
+  'success',
+  'alert',
+  'danger',
+  'info',
+  'neutral',
+  'surface',
+]
+
 export default meta
 
 export const Default: StoryFn = _args => (
-  <Chip>
-    Hello World!
-    <Chip.ClearButton onClick={() => console.log('clear')} />
-  </Chip>
+  <div className="flex flex-col items-start gap-md">
+    {designs.map(design => (
+      <Chip design={design} key={design}>
+        <Chip.Content>Hello World!</Chip.Content>
+        <Chip.ClearButton onClick={() => console.log(`clear ${design}`)} />
+      </Chip>
+    ))}
+  </div>
+)
+
+export const Intent: StoryFn = _args => (
+  <div className="flex flex-col flex-wrap gap-md">
+    Default
+    {designs.map(design => (
+      <div key={design} className="flex flex-wrap gap-md">
+        {intents.map(intent => (
+          <Chip design={design} key={`${design}-${intent}`} intent={intent}>
+            {intent} chip huge huge chip long and wrong
+          </Chip>
+        ))}
+      </div>
+    ))}
+    Actionable
+    {designs.map(design => (
+      <div key={design} className="flex flex-wrap gap-md">
+        {intents.map(intent => (
+          <Chip
+            design={design}
+            key={`${design}-${intent}`}
+            intent={intent}
+            onClick={() => console.log(`click ${design} ${intent}`)}
+          >
+            {intent} chip huge huge chip long and wrong
+          </Chip>
+        ))}
+      </div>
+    ))}
```

Comment:
WDTY is we only display the intent here ? the `Action` could be another story

Reply:
for sure. This is a porototype for testing it working on both usages. The PR for stories is not ready yet and will cover all the required properties variations